integration-tests/manager-integration-tests/manager-integration-tests-v2/src/test/java/com/redhat/service/smartevents/integration/tests/v2/steps/Hooks.java

```diff
@@ -60,85 +68,86 @@ public void before(Scenario scenario) {
         context.setStartTime(Instant.now());
     }
 
+    @After
+    public void cleanUp() {
+        boolean disabledCleanup = Boolean.parseBoolean(DISABLE_CLEANUP);
+        boolean disabledFailedCleanup = Boolean.parseBoolean(DISABLE_FAIL_CLEANUP);
+        boolean scenarioFailed = context.getScenario().isFailed();
+        if (disabledCleanup) {
+            context.getScenario().log("Cleanup disabled, all resources stays in the cluster");
+        } else if (disabledFailedCleanup && scenarioFailed) {
+            context.getScenario().log("Failed scenario cleanup disabled and the scenario failed, all resources stays in the cluster");
+        } else {
+            // Delete AWS SQS queues
+            //cleanAwsSQSQueues();
+
+            // Delete Kafka topics and related ACLs
+            //cleanKafkaTopics();
+
+            String token = BridgeUtils.retrieveBridgeToken();
+            // Remove all bridges/processors created
+            context.getAllBridges().values()
+                    .parallelStream()
+                    .filter(bridgeContext -> !bridgeContext.isDeleted())
+                    .forEach(bridgeContext -> {
+                        final String bridgeId = bridgeContext.getId();
+                        BridgeResponse bridge = BridgeResource.getBridgeDetails(token, bridgeId);
+                        // Skipping processor deletion steps for now as implementation of v2 API is still in progress for processor
```

Comment:
Looks to me like the processor deletion for v2 API is already implemented now, can you please re-check?

Reply:
Yeah sure

Reply:
@winklerm I think we can enable this once processor creation feature added..wdyt?

Reply:
@Srihari1192 It looks to me like this is already available: https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/manager-parent/manager-v2/src/main/java/com/redhat/service/smartevents/manager/v2/api/user/ProcessorsAPI.java#L160

But I might have missed something - do we need anything further to re-enable the processor deletion in the tests?

Reply:
Yes the processor delete changes in hook may be need some changes in V2 
@sakash20 Check for delete processor if there is any changes required and update in different PR

Reply:
Yeah. will start looking into processor part and raise the changes with different PR